conformance/1.7/README.md

```diff
@@ -0,0 +1,27 @@
+# Kubeflow conformance program (WIP)
+
+Before running the conformance tests, you need to configure kubectl default context to point to the k8s cluster that is hosting Kubeflow.
+
+TODO: Make the kubeflow namespace configurable.
+
+To run version <x.y> of the conformance test.
+
+`cd kubeflow/conformance/<x.y>`
+
+**Run conformance test**
+
+`make run`
```

Comment:
NIT: add brief comments on how to run component-specific tests (e.g. `make run-kfp`)

Reply:
Done.